WcaOnRails/app/views/competitions/_competition_info.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="col-md-6">
     <dl class="dl-horizontal compact">
       <dt><%= t '.date' %></dt>
-      <dd><%= wca_date_range(competition.start_date, competition.end_date) %></dd>
+      <dd><%= wca_date_range(competition.start_date, competition.end_date) %> (<%= link_to t('.add_to_calendar'), competition_download_ics_path(competition), format: :ics %>)</dd>
```

Comment:
As you mentioned in your comment you could use the calendar plus icon, combined with a tooltip when hovering the icon.
```ruby
 <%= link_to(icon("fab", "calendar-plus"), competition_path(competition, format: :ics),
           title: t('.add_to_calendar'),
           data: {
               toggle: "tooltip",
               placement: "top",
               container: "body",
            }) %>
```
(I'm not actually sure the `format: :ics` will work, but I think it will)